.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.activeBackground": "#d04649",
+        "activityBar.background": "#d04649",
+        "activityBar.foreground": "#e7e7e7",
+        "activityBar.inactiveForeground": "#e7e7e799",
+        "activityBarBadge.background": "#37cb34",
+        "activityBarBadge.foreground": "#15202b",
+        "commandCenter.border": "#e7e7e799",
+        "sash.hoverBorder": "#d04649",
+        "statusBar.background": "#b52e31",
+        "statusBar.foreground": "#e7e7e7",
+        "statusBarItem.hoverBackground": "#d04649",
+        "statusBarItem.remoteBackground": "#b52e31",
+        "statusBarItem.remoteForeground": "#e7e7e7",
+        "titleBar.activeBackground": "#b52e31",
+        "titleBar.activeForeground": "#e7e7e7",
+        "titleBar.inactiveBackground": "#b52e3199",
+        "titleBar.inactiveForeground": "#e7e7e799"
+    },
+    "peacock.color": "#b52e31"
+}
```

Comment:
I see a commit with the message `ignore .vsvode in git`. I suppose you forgot to include .gitignore/excluse .vscode/settings.json?

Reply:
yep, please remove this file from the git working tree and add an ignore rule in `.gitignore` :)

Reply:
Hehehe, looks like I need a break xD